### DISCUSSION TITLE
Mechanical prophylaxis for vte

### DISCUSSION THREAD

**Discussion by bbpob** [MIT-LCP/mimic-code#1515] [MIMIC-IV] [answered]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

**Comment by bbpob** [comment]
Hello, I want to extract information about patients receiving mechanical prophylaxis for VTE from MIMIC-IV. But I can't find that information. Does it not exist？

**Comment by alistairewj** [comment, marked as answer]
I believe I helped out over e-mail, so repeating it here. The following query gives some help: ```select * from `physionet-data.mimiciv_icu.d_items` where category = 'Treatments'```

For example there is itemid 224089 - Anti Embolic Device
